contracts/Registry.sol

```diff
@@ -23,12 +23,18 @@ contract Registry is FundsRecovery {
 
     address payable public dex;  // Any uniswap v2 compatible DEX router address
     uint256 public minimalHermesStake;
-    address internal channelImplementationAddress;
-    address internal hermesImplementationAddress;
+
+    struct Implementation {
```

Comment:
this should probably be "HermesImplementation" as it's ambiguous now

Reply:
It is about hermes and about channel...